includes/blocks/class-sensei-course-outline-block.php

```diff
@@ -217,7 +217,12 @@ function( $block ) {
 	 */
 	protected function render_lesson_block( $block ) {
 
-		$css = Sensei_Block_Helpers::build_styles( $block );
+		$css = Sensei_Block_Helpers::build_color_styles( $block );
+
+		if ( ! empty( $block['attributes']['fontSize'] ) ) {
+			$css['inline_styles'][] = sprintf( 'font-size: %spx', $block['attributes']['fontSize'] );
+		}
```

Comment:
I think this can be moved into the `build_styles` instead, since it basically does the same thing as the colors, and it's also something we will need for multiple blocks.

Reply:
Fixed in 61257b1c391f6c8ed58cfdc5c01b2a3cd6430871